play-json/shared/src/main/scala/Json.scala

```diff
@@ -6,39 +6,38 @@ package play.api.libs.json
 import java.io.InputStream
 
 /**
- * Helper functions to handle JsValues.
+ * @define jsonParam @param json the JsValue to convert
+ * @define returnStringRepr A String with the json representation
+ * @define readDescription Creates a `Reads[T]` by resolving, at compile-time, the case class fields or sealed family, and the required implicits.
+ * @define handlerTypeParam @tparam A the type for which the handler must be materialized
  */
-object Json {
-
+sealed trait JsonFacade {
```

Comment:
Would it be useful for this to be non-sealed? I could imagine it being useful to create a custom `JsonFacade` with special behavior (e.g. logging parsed values). Though I guess there are also risks to that.